modules/ROOT/pages/addition/agent-installation/self-registered.adoc

```diff
@@ -327,29 +349,24 @@ Ensure agent has contacted NOM server, is online and is reporting DBMS(s) correc
 . Return to Agents listing in global settings
 +
 image::agents.png[width=800]
-. An agent in standby mode shows up in the list of agents in NOM UI with `Unauthorized` status. To enable the agent to continue it's normal execution, agent needs to be approved from NOM UI as shown below:
+. An agent in standby mode shows up in the list of agents in NOM UI with `Unauthorized` status. 
+To enable the agent to continue it's normal execution, agent needs to be approved from NOM UI as shown below:
 +
 image::agent-unauthorized.png[width=800]
 . To approve the agent click on `...` agent action icon and click `Approve Agent`:
 +
 image::agent-action-menu.png[width=800]
 . An edit view to optionally update agent name or description will show as below:
 +
-NOTE: An unauthorized self-registering agent will have a default name derived from its host information. It's recommended to update this name or use the optional configuration described <<agent-meta-data,here>>.
-+
 image::agent-approve.png[width=800]
-. Once the agent is approved, the agent will receive initial configuration from NOM server which it persists on the instance host in a file named `nom-agent-config.yaml`.
-. Upon approval, the agent status changes to `Offline` until the agent connects to the server with the initial configuration.
+. Once the agent is approved (through UI or through mutual authentication), the agent will receive token information from NOM server which it persists on the instance host in a file named `nom-agent-config.yaml`.
+. Upon approval, the agent status changes to `Offline` until the agent connects to the server with the token information.
 +
 image::agent-approved-offline.png[width=800]
 . Once the agent is successfully connected to Server and reports the instances it needs to monitor, it's status changes to `Online`.
 +
 image::agent-approved-online.png[width=800]
 . Check that there is a value for _Last contact time_.
-    ** If the agent has never contacted NOM server then go back to where the agent is running and check the logs.
```

Comment:
I know that these are the same steps for manual installation, but they still apply to self-registration so kept them as they were. This is the duplication of docs I was referring to.

Reply:
I actually removed them because they made no sense if the user has just done the manual approval steps. I couldn't quite work out how best to handle the mutual authentication case elegantly...

Reply:
I'll try again 😄